test/expenses.routes.test.js

```diff
@@ -632,68 +565,95 @@ describe('expenses.routes.test.js', () => {
 
-                  it('THEN creates a transaction paid activity', () =>
-                    expectTransactionPaidActivity(group, host, transaction)
-                      .tap(activity => expect(activity.data.paymentResponse).to.deep.equal(payMock)));
-                });
-              });
+                    it('THEN marks expense as paid', () => expect(expense.status).to.be.equal('PAID'));
 
-              function expectTransactionCreated(expense, transaction) {
-                expect(transaction).to.have.property('netAmountInGroupCurrency', -12000);
-                expect(transaction).to.have.property('ExpenseId', expense.id);
-                // TODO remove #postmigration, info redundant with joined tables?
-                expect(transaction).to.have.property('amount', -expense.amount/100);
-                expect(transaction).to.have.property('currency', expense.currency);
-                expect(transaction).to.have.property('description', expense.title);
-                expect(transaction).to.have.property('status', 'REIMBURSED');
-                expect(transaction).to.have.property('UserId', host.id);
-                expect(transaction).to.have.property('GroupId', expense.GroupId);
-                // end TODO remove #postmigration
-              }
+                    it('THEN creates transaction', () => {
+                      expectTransactionCreated(expense, transaction);
+                      expect(transaction.PaymentMethodId).to.be.equal(paymentMethod.id);
+                    });
 
-              function expectTransactionPaidActivity(group, user, transaction) {
-                return Activity
-                  .findOne({ where: { type: 'group.transaction.paid' }})
-                  .tap(activity => {
-                    expect(activity.UserId).to.be.equal(user.id);
-                    expect(activity.GroupId).to.be.equal(group.id);
-                    expect(activity.TransactionId).to.be.equal(transaction.id);
-                    expect(activity.data.user.id).to.be.equal(user.id);
-                    expect(activity.data.group.id).to.be.equal(group.id);
-                    expect(activity.data.transaction.id).to.be.equal(transaction.id);
+                    it('THEN creates a transaction paid activity', () =>
+                      expectTransactionPaidActivity(group, host, transaction)
+                        .tap(activity => expect(activity.data.paymentResponse).to.deep.equal(payMock)));
                   });
-              }
+                });
+
+                function expectTransactionCreated(expense, transaction) {
+                  expect(transaction).to.have.property('netAmountInGroupCurrency', -12000);
+                  expect(transaction).to.have.property('ExpenseId', expense.id);
+                  // TODO remove #postmigration, info redundant with joined tables?
+                  expect(transaction).to.have.property('amount', -expense.amount/100);
```

Comment:
I'd rather not remove the tests, until we remove the actual code that's populating them.